Lunch/TodaysLunchViewController.swift

```diff
@@ -7,20 +7,150 @@
 //
 
 import UIKit
+import Parse
+import MapKit
+import GoogleMaps
 
-class TodaysLunchViewController: UIViewController {
+class TodaysLunchViewController: UIViewController, MKMapViewDelegate {
+
+    class CustomPointAnnotation: MKPointAnnotation {
+        var imageName: String!
+    }
+
+    @IBOutlet weak var selectedPlaceLabel: UILabel!
+    @IBOutlet weak var selectedVotesLabel: UILabel!
+    @IBOutlet weak var mapView: MKMapView!
 
-    @IBOutlet weak var todaysLunchLabel: UILabel!
-    
     var textForLunchLabel = ""
-    
+    var textForVotesLabel = ""
+    var selectedIndex = -1
+    var placesClient: GMSPlacesClient?
```

Comment:
Why not just put `var placesClient = GMSPlacesClient()` up here and avoid the optional?